css-grid-classes/styles.css

```diff
@@ -0,0 +1,66 @@
+/* body {
+  margin-left: 200px;
+}
+
+.section1,
+.section2,
+.section3,
+.section4 {
+  display: flex;
+}
+
+h1,
+h2,
+h3 {
+  font-weight: 700;
+}
+h1 {
+  font-size: 50px;
+}
+
+h2,
+h3 {
+  font-size: 30px;
+}
+#dk {
+  margin-left: 20px;
+}
+
+ .p1,
+ .p2,
+ .p3,
+ .p4,
+ .p5 {
+   line-height: 20px;
+   font-size: 20px;
+ }
+
+
+
+.section2 img {
+  width: 180px;
+ padding: 40px;
+}
+.section4 {
+  margin-top: 50px;
+} */
+
+body {
+  margin-left: 100px;
+}
+.column-one-third,
```

Comment:
remember, there should be only one `row` class set to display flex. And all of the column classes need to be given proper width values.